projects/igniteui-angular/src/lib/date-common/util/date-time.util.spec.ts

```diff
@@ -233,6 +236,30 @@ describe(`DateTimeUtil Unit tests`, () => {
         expect(result).toEqual('MM/dd/yyyy');
     });
 
+    it('should properly build input formats based on locale for dateTime data type ', () => {
+        spyOn(DateTimeUtil, 'getDefaultInputFormat').and.callThrough();
```

Comment:
PS: This spy is a bit useless :) Noticed it came from the prev test, but still :)